src/main/java/io/strimzi/test/container/KafkaVersionService.java

```diff
@@ -197,7 +197,7 @@ public String toString() {
      */
     public KafkaVersion latestRelease() {
         // at least one release in the json schema is needed
-        if (this.logicalKafkaVersionEntities == null || this.logicalKafkaVersionEntities.size() < 1) {
+        if (this.logicalKafkaVersionEntities.isEmpty()) {
```

Comment:
The field is `final` and initialized when declared so it should never be `null`.